roles/ipxe/tasks/main.yml

```diff
@@ -58,12 +61,12 @@
     chdir: "{{ ansible_env.HOME }}/ipxe/src"
```

Comment:
You might want to move this to go in a subdir of the build-space instead of the users $HOME. Everything else goes to /tmp/on-imagebuilder/its-working-dir, but this one plops itself down elsewhere. There is already an ipxe_build_artifact_path defined in hosts, so adding an ipxe_build_working_path to another directory within the build-space is pretty simple. Aside from just being odd having one part of the work done in a different place than the others, it really bites on a mounted home directory :)